### ISSUE TITLE
some problem for trainning

### ISSUE THREAD

**Issue by linchaozhang** [baileyqbb/pva-faster-rcnn#1]
Hello, I try to train a model for 2 classes(person and background), loss descend to 0.1, but result of test is so bad, mAP is only 15%. i don't know where is the problem

**Comment by linchaozhang** [comment]
@baileyqbb

**Comment by baileyqbb** [comment]
@linchaozhang Hi! Thanks for following the branch! 
Can you give more details of your training steps? Train from scratch or fine-tune from the pre-trained models? Which dataset used for training? How much steps have you trained for your model? 
Besides, if possible, could you provide the plot of the loss using tensorboard?
